### PR TITLE
Added listener for when selecting a participant fails +fallback toast

### PR DESCRIPTION
**Pull request by thecombatwombat**

### CODE REVIEW

File: layer-atlas/src/main/java/com/layer/atlas/AtlasAddressBar.java [hunk 1]
```diff
@@ -743,4 +758,8 @@ public ViewHolder(ViewGroup parent) {
     public interface OnParticipantSelectionChangeListener {
         void onParticipantSelectionChanged(AtlasAddressBar conversationLauncher, List<Identity> participants);
     }
+
+    public interface OnParticipantSelectionFailedListener {
+        void onParticipantSelectionFailed (String reason);
```
Comment (thecombatwombat) [inline layer-atlas/src/main/java/com/layer/atlas/AtlasAddressBar.java]: Okay, yeah that makes sense... Lemme look up a usage...

**Comment by thecombatwombat** [comment]
@smpete @sriamar ready for you guys to take another look...

File: layer-atlas/src/main/java/com/layer/atlas/AtlasAddressBar.java [hunk 2]
```diff
@@ -47,11 +48,14 @@
 import java.util.Set;
 
 public class AtlasAddressBar extends LinearLayout {
+    public static final int MAX_PARTICIPANTS = 25;
```
Comment (sriamar) [inline layer-atlas/src/main/java/com/layer/atlas/AtlasAddressBar.java]: Ideally, this should come from SDK. File a ticket in JIRA to track that.

Comment (thecombatwombat) [inline layer-atlas/src/main/java/com/layer/atlas/AtlasAddressBar.java]: 👍

**Pull request merged**